roles/openshift/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Common variables
+openshift_env: 'test'
+openshift_roles: [ 'master', 'node' ]
+openshift_bind_ip: "{{ ansible_default_ipv4.address }}"
+openshift_public_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
I was looking for a sane default that would work outside of cloud environments.  I figured for cloud environments we'd inform the user that they must choose the appropriate variable set from their dynamic inventory as a role argument or would have to set the variable as a host or inventory variable.